src/main/webapp/js/constants/entity-types.js

```diff
@@ -9,3 +9,4 @@ export const PERMISSION_ENTITY = 'permission';
 export const VISIT_TYPE_ENTITY = 'visitType';
 export const CAMPAIGN_MESSAGE_ENTITY = 'campaignMessage';
 export const USER_ENTITY = 'user';
+export const APP_SETTINGS_ENTITY = 'appSettings';
```

Comment:
I think you can remove it, because it will not be used
also it's not a typical entity